PureDOOM.h

```diff
@@ -189,8 +189,7 @@ void doom_set_getenv(doom_getenv_fn getenv_fn);
 void doom_init(int argc, char** argv, int flags);
 
 // Call this every frame
-void doom_update(); // This will update at 35 FPS
-void doom_force_update(); // This will run a frame everytime it's called, regardless of FPS.
```

Comment:
Why remove my code?

Reply:
I realize what happened here. You regenerated the PureDOOM.h without rebasing latest :)

Reply:
I didn't regenerate PureDOOM.h. I edited PureDOOM.h and then changed the source files.